src/common/utility_unix.cpp

```diff
@@ -71,12 +72,24 @@ void setLaunchOnStartup_private(const QString &appName, const QString &guiName,
             qCWarning(lcUtility) << "Could not write auto start entry" << desktopFileLocation;
             return;
         }
+        // When running inside an AppImage, we need to set the path to the
+        // AppImage instead of the path to the executable
+        QString executablePath;
+        QString appImagePath = qEnvironmentVariable("APPIMAGE");
+        if (!appImagePath.isNull() && QFile::exists(appImagePath)) {
```

Comment:
Optional but since I'm a const freak I'd advise going for the ternary operator so that executablePath can be made const, it looked something like that I think:
```
const auto runningInsideAppImage = !appImagePath.isNull() && QFile::exists(appImagePath);
const auto executablePath = runningInsideAppImage ? appImagePath : executablePath;
```

Reply:
Actually a nice idea, b.c. it gets rid of the uninitialized variable, which I do not like.

Reply:
Well that's also why I tend to auto more as well, it prevents uninitialized variables. ;-)